addons/events/script_component.hpp

```diff
@@ -33,7 +33,9 @@
 
 #define CALL_EVENT(params,event) {\

```

Comment:
Should the macro argument "params" be renamed so as to not collide with the reserved word (command) `params`?